src/app/index.html

```diff
@@ -0,0 +1,5 @@
+<h2>Examples</h2>
```

Comment:
It seems like we should have visual tests for this. We should probably change this to be `Visual tests` to match our other repos when we do that.